test_scripts/API/SetAudioStreamingIndicator/037_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_media_app_RPC_disallowed_by_user.lua

```diff
@@ -0,0 +1,190 @@
+  end
+
+  data.policy_table.functional_groupings["Base-4"].rpcs.SetAudioStreamingIndicator = nil
+  data.policy_table.functional_groupings["Testing_group"] = {
+    user_consent_prompt = "Purpose_of_Test",
+    rpcs = {
+      SetAudioStreamingIndicator = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" }
+      }
+    }
+  }
+  local data_preloaded = commonFunctions:cloneTable(data)
+
+  data_preloaded = json.encode(data_preloaded)
+  file = io.open(config_path .. 'sdl_preloaded_pt.json', "w")
+  file:write(data_preloaded)
+  file:close()
+
+  data.policy_table.module_config.preloaded_pt = nil
+  data.policy_table.module_config.preloaded_date = nil
+  data.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] =
+  {
+    keep_context = false,
+    steal_focus = false,
+    priority = "NONE",
+    default_hmi = "NONE",
+    groups = {"Base-4", "Testing_group"}
+  }
+
+  data = json.encode(data)
+  file = io.open("SetAudioStreamingIndicator_group1.json", "w")
+  file:write(data)
+  file:close()
+end
+
+
+--[[ General Precondition before ATF start ]]
+create_ptu_SetAudioStreamingIndicator_group()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_trigger_getting_device_consent()
+  testCasesForPolicyTable:trigger_getting_device_consent(self, config.application1.registerAppInterfaceParams.appName, config.deviceMAC)
+end
+
+function Test:Precondition_SetAudioStreamingIndicator_DISALLOWED_audioStreamingIndicator_PLAY_PAUSE()
+  local corr_id = self.mobileSession:SendRPC("SetAudioStreamingIndicator", { audioStreamingIndicator = "PLAY_PAUSE" })
+
+  EXPECT_HMICALL("UI.SetAudioStreamingIndicator", {}):Times(0)
+  EXPECT_RESPONSE(corr_id, { success = false, resultCode = "DISALLOWED"})
+end
+
+function Test:Precondition_PTU_appPermissionsConsentNeeded_false()
+  local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+  EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
+  :Do(function(_,_)
+    self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest", { requestType = "PROPRIETARY", fileName = "filename"})
+
+    EXPECT_NOTIFICATION("OnSystemRequest", { requestType = "PROPRIETARY" })
+    :Do(function(_,_)
+      local CorIdSystemRequest = self.mobileSession:SendRPC("SystemRequest", { fileName = "PolicyTableUpdate", requestType = "PROPRIETARY"}, "SetAudioStreamingIndicator_group1.json")
+
+      EXPECT_HMICALL("BasicCommunication.SystemRequest")
+      :Do(function(_,data)
+        self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate", { policyfile = "/tmp/fs/mp/images/ivsu_cache/PolicyTableUpdate"})
```

Comment:
@istoimenova, For '/tmp/fs/mp/images/ivsu_cache/' please use parameter from .ini file
Ex: commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath")